tools/gulp/tasks/build.ts

```diff
@@ -69,16 +69,16 @@ task(':library:schematics:compile', (done: (err?: string) => void) => {
 });
 
 /** Copies the migration json file over to the dist folder for the schematics */
-task(':library:schematics:copy-json', () =>
-  src(join(buildConfig.libDir, 'schematics/migration.json')).pipe(
-    dest(join(buildConfig.libOutputDir, 'schematics')),
-  ),
-);
+// task(':library:schematics:copy-json', () =>
+//   src(join(buildConfig.libDir, 'schematics/migration.json')).pipe(
+//     dest(join(buildConfig.libOutputDir, 'schematics')),
+//   ),
+// );
```

Comment:
If we don't need it any more, please remove the code. 
If we still need the task, please comment it in again.